.github/wiki/Setup-More-Available-Configs.md

```diff
@@ -0,0 +1,6 @@
+# More available configs
+| Parameter name | Is mandatory | Description                                                                     |
+| :------------- | :----------: | :------------------------------------------------------------------------------ |
+| APP_VERSION    |      No      | The app version that is set when initialize the project. Default is `0.1.0`     |
+| BUILD_NUMBER   |      No      | The build number that is set when initialize the project. Default is `1`        |
+| json_serializable.field_rename   |      No      | Defines the automatic naming strategy when converting class field names into JSON map keys. <br/>Available options: <br/> - `none`: Use the field name without changes.<br/> - `kebab`: Encodes a field named `kebabCase` with a JSON key `kebab-case`.<br/> - `snake`: Encodes a field named `snakeCase` with a JSON key `snake_case`.<br/> - `pascal`: Encodes a field named `pascalCase` with a JSON key `PascalCase`.        |
```

Comment:
@markgravity Should it be `Use the field names without changing`? 🤔

Reply:
@doannimble I just copy & paste from their document 😆 
![Screen Shot 2022-06-03 at 14 37 05](https://user-images.githubusercontent.com/17875522/171809698-c630ad94-4e85-4bf1-9d8f-bb4e84cee8c7.png)